core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java

```diff
@@ -1,2 +1,145 @@
-package grep.src.main.java.ca.jrvs.apps.grep;public class JavaGrepImp {
+package ca.jrvs.apps.grep;
+
+import org.apache.log4j.BasicConfigurator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
```

Comment:
It is better to have individual imports, to know exactly what is being user from a library, and also to import only what is needed.